src/locale/languages/en.js

```diff
@@ -157,6 +157,9 @@ export const en = {
     followingList: {
       title: 'Following',
     },
+    followYou: {
+      title: 'Follows you',
+    },
```

Comment:
@housseindjirdeh no need to do that for the `en.js` file, this is only for translators to keep some kind of internal record.

+1 on @andrewda request, we need a fallback to english and developers shouldn't have to touch translations files. It used to be the case with the previous system.

@reyhansofian if you do touch the french file, here's the translation: "Vous suit".